go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/BTrDB/btrdb/v5
+
+go 1.14
+
+require (
+	github.com/BTrDB/btrdb v5.9.1+incompatible
```

Comment:
Yeh we should ensure this doesn't happen

Reply:
removed after refactoring insert_test.go